daycare/models.py

```diff
@@ -18,7 +21,14 @@ class DayCare(models.Model):
     address = models.CharField(max_length=50, blank=True, validators=[MaxLengthValidator])
 
     def __str__(self):
-        return self.name
+        return f'{self.name} name'
```

Comment:
Why should it return the name of the daycare, following the word "name", such as "DAYCARE-NAME name"?
Wouldn't it be better with "Daycare name: {self.name}", for example?

Reply:
Your way seems logical overall, I don't think it is substantial but I will change it after all.